app/src/main/java/lt/vilnius/tvarkau/utils/GlobalConsts.java

```diff
@@ -19,6 +19,7 @@
     public final static String CODE_FOR_VILNIUS_EMAIL = "tvarkau.vilniu@codeforvilnius.lt";
     public final static String VILNIUS_MUNICIPALITY_EMAIL = "e.vicemeras@vilnius.lt";
     public final static String VILNIUS_MUNICIPALITY_PHONE = "1664";
+    public final static String GOOGLE_PLAY_SERVICES = "https://play.google.com/store/apps/details?id=com.google.android.gms";
```

Comment:
I think you can use this field directly in `NewProblemActivity` if you are not going to use it any other place.